bin/dump-schema

```diff
@@ -13,4 +14,6 @@ mysqldump "$@" \
    pull_labels \
    pull_signatures \
    pulls \
-   > schema.sql
+   > $SCHEMA_FILE
+
+sed -i -e "s/CREATE TABLE/CREATE TABLE IF NOT EXISTS/" $SCHEMA_FILE
```

Comment:
Huh, I guess there [isn't a mysqldump flag to do this](https://stackoverflow.com/questions/2220469/create-table-if-not-exists-from-mysqldump).